db/schema.rb

```diff
@@ -659,6 +659,15 @@
   add_index "organisation_visits", ["organisation_id"], name: "index_organisation_visits_on_organisation_id", using: :btree
   add_index "organisation_visits", ["visit_id", "organisation_id"], name: "index_organisation_visits_on_visit_id_and_organisation_id", unique: true, using: :btree
 
+  create_table "specifics", force: :cascade do |t|
```

Comment:
I think it would be good to talk through why we want discourage schema
changes with plugins. Also I want to discuss downsides of a shared model..
maybe there are none but It kind of worries me. I'll see you tomorrow!

On Wed, 17 Feb 2016 at 14:21 James Kiesel <notifications@github.com> wrote:

> In db/schema.rb
> <https://github.com/loomio/loomio/pull/2863#discussion_r53108695>:
>
> > @@ -659,6 +659,15 @@
> >    add_index "organisation_visits", ["organisation_id"], name: "index_organisation_visits_on_organisation_id", using: :btree
> >    add_index "organisation_visits", ["visit_id", "organisation_id"], name: "index_organisation_visits_on_visit_id_and_organisation_id", unique: true, using: :btree
> >
> > +  create_table "specifics", force: :cascade do |t|
>
> Happy to take em out for this PR, but we'll need em for reactions, and
> ideally we'd move webhooks to working this way as well. I kinda want to
> discourage schema changes in plugins if we can help it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2863/files#r53108695>.
>